### PR TITLE
Add PlatON mainnet

### PR DESCRIPTION
**Pull request by benbaley**
Add PlatON mainnet

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `PlatON` more than once

**Comment by benbaley** [comment]
> short name must be unique - but found `PlatON` more than once



> short name must be unique - but found `PlatON` more than once

I have changed the `shortname` of PlatON testnet to 'platondev' and kept the shortname of PlatON mainnet, will this work please?

**Review by ligi — CHANGES_REQUESTED** [review]
The chainId in the filename does not match the chainID in the json

File: _data/chains/eip155-210309.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "decimals": 18
   },
   "infoURL": "https://www.platon.network",
-  "shortName": "PlatON",
+  "shortName": "platondev",
```
Comment (mohamadrezakheradmand) [inline _data/chains/eip155-210309.json:18]: platondev

File: _data/chains/eip155-210309.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
     "https://devnetopenapi.platon.network/rpc",
 	"wss://devnetopenapi.platon.network/ws"
   ],
-  "faucets": [],
```
Comment (mohamadrezakheradmand) [inline _data/chains/eip155-210309.json]: شیر الات

**Review by mohamadrezakheradmand — COMMENTED** [review]
شیرالات

File: _data/chains/eip155-210309.json (hunk 1 above)
Comment (mohamadrezakheradmand) [reply _data/chains/eip155-210309.json:18]: platondev

**Pull request closed without merging**